samples/custom_dispatch/xnnpack/plugin/system_plugin.c

```diff
@@ -132,6 +138,112 @@ static int mul2_1d_workgroup(void* params_ptr, void* context, void* reserved) {
   return 0;
 }
 
+// TODO(ramiro050): refactor common logic with multiply_2 above
+static int batch_matrix_multiply_workgroup(void* params_ptr, void* context,
+                                           void* reserved) {
+  system_plugin_t* plugin = (system_plugin_t*)context;
+  typedef struct {
+    const float* restrict binding0;
+    size_t binding0_offset;
+    size_t binding0_stride0;
+    size_t binding0_stride1;
+    size_t binding0_stride2;
+    const float* restrict binding1;
+    size_t binding1_offset;
+    size_t binding1_stride0;
+    size_t binding1_stride1;
+    size_t binding1_stride2;
+    float* restrict binding2;
+    size_t binding2_offset;
+    size_t binding2_stride0;
+    size_t binding2_stride1;
+    size_t binding2_stride2;
+    size_t binding0_size0;
```

Comment:
nit: put the size next to the same binding just like the offset and strides.

Reply:
It's unclear to me how to do this. Every tensor operand when using the `ukernel` flow turns into `tensor -> base_ptr, offset, *strides`. One can then *append* any extra arguments to the `ukernel.generic` call. This is why the sizes are towards the end.